### PR TITLE
ui: add usage instructions for PAT

### PR DESCRIPTION
**Pull request by LJea**
### Added usage instructions for PAT
Added supplementary instructions for using PAT
In some applications, you must enter a password when logging in to GIT, which makes pat unusable. Add supplementary instructions to make it more correct to use PAT
Link to the issue: n/a

### Checklist

- [x] I agree to follow the [Code of Conduct](https://go.dev/conduct) by submitting this pull request.
- [x] I have read and acknowledge the [Contributing guide](https://github.com/gogs/gogs/blob/main/.github/CONTRIBUTING.md).
- [x] I have added test cases to cover the new code.

### CODE REVIEW

**Comment by LJea** [comment]
I rolled back changes other than conf/locale_en-US.ini

File: conf/locale/locale_en-US.ini [hunk 1]
```diff
@@ -370,6 +370,7 @@ two_factor_disable_success = Two-factor authentication has disabled successfully
 manage_access_token = Manage Personal Access Tokens
 generate_new_token = Generate New Token
 tokens_desc = Tokens you have generated that can be used to access the Gogs APIs.
+use_pat_token=The username you log into the Git app with is your personal action token You can log in with an empty password or x-oauth-basic as the password.
```
Comment (unknwon) [inline conf/locale/locale_en-US.ini:373]:
```suggestion
access_token_tips=The username you log into the Git app with is your personal access token. You can log in with an empty password or "x-access-token" as the password.
```

File: templates/user/settings/security.tmpl [hunk 2]
```diff
@@ -23,7 +23,8 @@
 				{{if .TwoFactor}}
 					<br>
 					<p>{{.i18n.Tr "settings.two_factor_view_recovery_codes" AppSubURL "/user/settings/security/two_factor_recovery_codes" | Safe}}</p>
-					<p>{{.i18n.Tr "settings.two_factor_http" AppSubURL "/user/settings/applications" "https://{token}@try.gogs.io/user/repo.git" | Safe}}</p>
+					<p>{{.i18n.Tr "settings.two_factor_http" AppSubURL "/user/settings/applications" "https://{token}:x-oauth-basic@try.gogs.io/user/repo.git" | Safe}}</p>
```
Comment (unknwon) [inline templates/user/settings/security.tmpl:26]:
```suggestion
					<p>{{.i18n.Tr "settings.two_factor_http" AppSubURL "/user/settings/applications" "https://{token}:x-access-token@try.gogs.io/user/repo.git" | Safe}}</p>
```

File: templates/user/settings/security.tmpl [hunk 3]
```diff
@@ -23,7 +23,8 @@
...
+					<p>{{.i18n.Tr "settings.use_pat_token"}}</p>
```
Comment (unknwon) [inline templates/user/settings/security.tmpl:27]:
```suggestion
					<p>{{.i18n.Tr "settings.access_token_tips"}}</p>
```

**Review by unknwon — APPROVED** [review]
Thanks for the PR!

**Pull request merged**